.github/workflows/flutter-build.yml

```
@@ -756,20 +841,200 @@ jobs:
+
+      - name: Verify update metadata
+        shell: bash
+        env:
+          VERSION: ${{ env.VERSION }}
+          TAG_NAME: ${{ env.TAG_NAME }}
+          RUSTDESK_UPDATE_ED25519_PUBLIC_KEY: ${{ secrets.RUSTDESK_UPDATE_ED25519_PUBLIC_KEY }}
+        run: |
+          python res/generate_update_metadata.py verify --metadata ./rustdesk-update.json --signature ./rustdesk-update.json.sig --artifact "./artifacts/windows-x86_64/rustdesk-${VERSION}-x86_64.exe" --artifact "./artifacts/windows-x86_64/rustdesk-${VERSION}-x86_64.msi" --artifact "./artifacts/windows-x86/rustdesk-${VERSION}-x86-sciter.exe" --artifact "./artifacts/macos-aarch64/rustdesk-${VERSION}-aarch64.dmg" --artifact "./artifacts/macos-x86_64/rustdesk-${VERSION}-x86_64.dmg" --package-id rustdesk --version "$VERSION" --release-id "$TAG_NAME" --trusted-public-key-base64 "$RUSTDESK_UPDATE_ED25519_PUBLIC_KEY" --trusted-public-key-id 2026-ed25519-main
+
+      - name: Publish signed update release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.12)

[error] the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)